vllm/transformers_utils/tokenizer.py

```diff
@@ -29,7 +29,7 @@ def get_cached_tokenizer(
     tokenizer_all_special_tokens_extended = (
         tokenizer.all_special_tokens_extended)
     tokenizer_all_special_tokens = set(tokenizer.all_special_tokens)
-    tokenizer_len = len(tokenizer)
```

Comment:
why is this change necessary? `len(tokenizer)` might also contain special tokens, while `tokenizer.vocab_size` might not.

Reply:
this change has been reverted